src/sdk/resources/personalDrive.ts

```diff
@@ -0,0 +1,18 @@
+import { IHttpClient } from '../http';
+import { PagedResource } from './resource';
+import { PersonalDriveMatchesResource } from './personalDriveMatches';
+import { PersonalDriveExceptionsResource } from './personalDriveExceptions';
+
+export class PersonalDriveResource extends PagedResource<Account> {
+	constructor(httpClient: IHttpClient) {
+		super(httpClient, 'personal_drive');
+	}
+
+	matches(jobId: string): PersonalDriveMatchesResource {
+		return new PersonalDriveMatchesResource(this.httpClient, jobId);
+	}
+
+	exceptions(personalDriveJobId: string): PersonalDriveExceptionsResource {
```

Comment:
Why different parameter names for the same thing between this method and the one above?

Reply:
copy and paste.  fix it now, though.